### PR TITLE
Add aliyundrive

### PR DESCRIPTION
**Pull request by chyroc**
## Description

for: https://www.aliyundrive.com

## Screencast

image

list files:

image

share file:
image

create folder:

image

rename:

image

delete:

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/aliyundrive/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "name": "aliyundrive",
+  "title": "aliyundrive",
+  "description": "aliyundrive extension",
+  "icon": "aliyundrive.png",
+  "author": "chyroc",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "file_list",
+      "title": "AliyunDrive My Files",
```
Comment (pernielsentikaer) [inline extensions/aliyundrive/package.json:12]:
```suggestion
      "title": "My Files",
      "subtitle": "AliyunDrive",
```

File: extensions/aliyundrive/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "description": "list my files",
+      "mode": "view"
+    },
+    {
+      "name": "search_file",
+      "title": "AliyunDrive Search Files",
```
Comment (pernielsentikaer) [inline extensions/aliyundrive/package.json:18]:
```suggestion
      "title": "Search Files",
      "subtitle": "AliyunDrive",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @chyroc 👋 

Thanks for your contribution! 💪 

I'm not quite sure how to test/(and use) it, could you maybe provide a README so it's possible to understand what's happening, there are no preferences at all.

The extension suffers from [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states) - could you resolve this also?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by fe9lix** [comment]
Hey guys, I'm closing this PR because of inactivity. Feel free to reopen it again later and address the feedback.

**Pull request closed without merging**